### ISSUE TITLE
I get "Warning: Unknown prop `option`" when passing an array of string values as options

### ISSUE THREAD

**Issue by bigwillstyle4ever** [ericgio/react-bootstrap-typeahead#322]
Not sure about that one, I have been using the typeahead for a while without issues. Then left my project for a couple of week on the side, now have set it up again, I get this warning from React:

Warning: Unknown prop `option` on <a> tag. Remove this prop from the element:

2.4 Version

Code:
```jsx
const data = ['foo','bar'];

<Typeahead
  onInputChange={(e) => this.handleInputChange(e)}
  options={data}
/>
```

Stacktrace:
```
    in a (created by BaseMenuItem)
    in li (created by BaseMenuItem)
    in BaseMenuItem (created by menuItemContainer(BaseMenuItem))
    in menuItemContainer(BaseMenuItem) (created by TypeaheadMenu)
    in ul (created by BaseMenu)
    in BaseMenu (created by Menu)
    in Menu (created by TypeaheadMenu)
    in TypeaheadMenu (created by Typeahead)
    in Overlay (created by Typeahead)
    in div (created by Typeahead)
    in Typeahead (created by WrappedTypeahead)
    in WrappedTypeahead (created by WrappedTypeahead)
    in WrappedTypeahead (created by Typeahead)
    in Typeahead (created by OnClickOutside(Typeahead))
    in OnClickOutside(Typeahead) (created by AddNewCustomerView)
```

**Comment by ericgio** [comment]
It looks like `option` is being passed down from `menuItemContainer`, which it shouldn't. Thanks for the report.

**Comment by ericgio** [comment]
This should be fixed as of v2.5.0

**Comment by bigwillstyle4ever** [comment]
You're welcome, no problems anymore. Good work!
